src/cryptography/x509.py

```diff
@@ -976,6 +982,11 @@ def __ne__(self, other):
         return not self == other
 
 
+class SubjectAlternativeName(GeneralNames):
```

Comment:
You can write the `get_values_for_type` as `self._general_names.get_values_for_type`, but otherwise it looks good to me.